.reuse/dep5

```diff
@@ -0,0 +1,23 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: Astarte Device SDK ESP32
+Source: https://github.com/astarte-platform/astarte-device-sdk-esp32
+
+Files: doc/docmain.dox
+Copyright: 2023 SECO Mind Srl
+License: Apache-2.0
+
+Files: examples/**/README.md
+Copyright: 2023 SECO Mind Srl
+License: Apache-2.0
+
+Files: examples/**/interfaces/*.json
+Copyright: 2023 SECO Mind Srl
+License: Apache-2.0
```

Comment:
For the json files use `CC0-1.0`
```suggestion
License: CC0-1.0
```

Reply:
Why creative commons? CC0 is only in this repos for the REUSE workflow. All the rest is licensed under LGPL or Apache 2.0.

Reply:
Keep `Apache-2.0`.